backend/src/useCases/users.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  OAuthUser,
+  User,
+  userFromOAuth,
+  userFromTable,
+} from "../models/index.js";
+import { usersRepository } from "../repositories/index.js";
+
+const updateUserHandle = async (
```

Comment:
I think we should add some input validation for handle, trimming, excluding problematic symbol, length min/max, ...